modules/Kconfig.tfm

```diff
@@ -20,12 +20,6 @@ config TFM_BOARD
 
 menuconfig BUILD_WITH_TFM
 	bool "Build with TF-M as the Secure Execution Environment"
-	select CMSIS_RTOS_V2
```

Comment:
If it's truly not needed, you can remove the NUM_PREEMPT_PRIORITIES below as well.

Reply:
Has this been tested with the two TF-M samples? I'll do so myself later today, but wanted to check if someone has already verified this isn't required any longer since it previously was for IPC support (circa TFM 1.1).

Reply:
Yeap, I did that for mps2 an521.